src/glossary.rst

```diff
@@ -5357,6 +5357,15 @@ struct value
 :dp:`fls_YmZfW9kWlbIX`
 A :dt:`struct value` is a :t:`value` of a :t:`struct type`.
 
+.. _fls_P7920ALJisrH:
+
+structurally equal
```

Comment:
After reading all the changes, I think we should discuss the term "structurally equal" and its semantics.